### ISSUE TITLE
Error in Eq.(1)?

### ISSUE THREAD

**Issue by chenkang455** [BIT-Vision/RSIR#1]
![image](https://github.com/BIT-Vision/RSIR/assets/72788314/39363755-36f1-4e1a-ae0d-f4ac98519ce7)
V_{DDpix} should be placed before the integral formula ?
@LonsonZheng

**Comment by LonsonZheng** [comment]
Yes, thank you for your reminder. We will make corrections and update the version on arXiv.
